node/src/server-modules/GlideJson.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+import { DecoderOption, GlideString } from "../BaseClient";
+import { ConditionalChange } from "../Commands";
+import { GlideClient } from "../GlideClient";
+import { GlideClusterClient } from "../GlideClusterClient";
+
+export type ReturnTypeJson = GlideString | GlideString[];
```

Comment:
```suggestion
export type ReturnTypeJson = GlideString | (GlideString | null )[];
```

Reply:
I'd rename this. It is return type for `json.get` only. The current name is too broad I think.
Actually, you don't have to define a new type there - you can define your function to return `Promise<GlideString | whatever>`